catroid/src/main/java/org/catrobat/catroid/content/actions/GoToRandomPositionAction.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2022 The Catrobat Team
```

Comment:
Can you just update this to 2010-202**4**.  Please also update the year for every file you change in the future :)